packages/test/test-end-to-end-tests/src/test/orderSequentially.spec.ts

```diff
@@ -28,7 +28,9 @@ const cellId = "cellKey";
 const mapId = "mapKey";
 
 describeCompat("Multiple DDS orderSequentially", "NoCompat", (getTestObjectProvider, apis) => {
-	const { SharedMap, SharedDirectory } = apis.dds;
+	const { SharedMap, SharedDirectory, SharedString } = apis.dds;
+	const { SequenceDeltaEvent } = apis.dataRuntime.packages.sequence;
```

Comment:
This test suite is IMO a great example of why I think this work matters: it does `instanceof` checks on `SequenceDeltaEvent`. The tests will therefore fail once more than no-compat is enabled (since there will be different SequenceDeltaEvent class declarations), but that's pretty subtle when everything is statically imported with some behind-the-scenes oongaboonga to load a different sequence package.

The tests will still fail as written for cross-version compat (you'd need to know whether to use the SequenceDeltaEvent from dataRuntime or from dataRuntimeForLoading)... but that's a problem we can solve later, and specifying things this way should hopefully make it more clear to test authors what's going on.